test/docker/executeContainerCommand.test.js

```diff
@@ -12,7 +13,11 @@ test('response includes stdout, stderr, and inspectOutput with ExitCode 0', asyn
     t.is(stderr.toString('utf8'), '');
     t.is(inspectOutput.ExitCode, 0);
   } finally {
-    // don't wait for container to stop since it could take a while
```

Comment:
Is this not true anymore?

Reply:
I haven't seen it take long, but I have seen that the process can exit before the command is sent to docker, and so the container never stops.